content/services/current_AARCH64_projects.rst

```diff
@@ -9,7 +9,7 @@ Below are a list of currently hosted AARCH64 projects
 `FOSS Projects`_
 
 .. _`FOSS Projects`:
-.. csv-table:: FOSS Projects (14 projects)
+.. csv-table:: FOSS Projects (15 projects)
```

Comment:
Looks like this actually needs to be 16

Reply:
It's actually only 15. AlmaLinux's description is split into 2 lines in the file.